corehq/apps/hqadmin/management/commands/stale_data_in_es.py

```diff
@@ -85,7 +85,8 @@ def handle(self, domain, data_models, delimiter, **options):
             print('Running for all SQL domains (and excluding Couch domains!)', file=sys.stderr)
 
         def print_data_row(data_row):
-            print(delimiter.join(data_row))
+            # Casting as str print `None` as 'None' and datetimes as 'YYYY-MM-DD hh-mm-ss'
+            print(delimiter.join(map(str, data_row)))
```

Comment:
was it not necessary to also modify the republish command to handle the new date format?

Reply:
It wasn't because `republish_doc_changes` just throws away the two datetime columns; it only cares about doc id, doc type, doc subtype, and domain